tests/Common/BranchComponentTest.php

```diff
@@ -171,19 +171,64 @@ function ($branch) use ($branchName) {
         $mainComponentDetail  = $components->getConfiguration($componentId, 'main-1');
         $this->assertSame('main-1', $mainComponentDetail['id']);
 
+        // add two config rows to test rowsSortOrder
+        $branchComponents->addConfigurationRow(
+            (new ConfigurationRow($configurationOptions))
+                ->setName('Dev 1 Row 1')
+                ->setRowId('dev-1-row-1')
+        );
+
+        $configurationOptions->setRowsSortOrder(['main-1-row-1', 'dev-1-row-1']);
```

Comment:
Nebylo by lepší nastavit sort order opačně, než je default? Nebo jsou by default od nejnovějšího?

Reply:
Aha, ono je to defaultně tak, že je to podle ID - takže nejdřív dev pak main podle abecedy :D